device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/MultiplexingClient.java

```diff
@@ -459,7 +460,7 @@ public void registerDeviceClients(Iterable<DeviceClient> deviceClients, long tim
 
                 if (deviceClientToRegister.getDeviceIO() != null && deviceClientToRegister.getDeviceIO().isOpen() && !deviceClientToRegister.isMultiplexed)
                 {
-                    throw new IllegalStateException("Cannot register a device client to a multiplexed connection when the device client was already opened.");
```

Comment:
All the other exceptions like this one threw UnsupportedOperationException, so it seemed appropriate for me to change this one to match